packages/http-sdk/package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "format": "prettier --write ./*.{ts,json} **/*.{ts,json}",
     "lint": "eslint .",
-    "update-apps-local-deps": "mvm-update"
+    "update-apps-local-deps": "mvm-update",
+    "validate:types": "tsc --noEmit && echo"
```

Comment:
note: `&& echo` at the end to ignore staged files which `lint-staged` passes to command. This is a temporary thing and after migration this npm command will be removed